content/docs/guides/vercel-managed-integration.md

```diff
@@ -0,0 +1,174 @@
+- Optionally creates a dedicated database branch for every Preview Deployment so you can test schema changes safely
+
+<Admonition type="note" title="Who should use this path?">
+Choose the Vercel-Managed Integration if you **do not already have a Neon account** *or* you prefer to consolidate payment for Neon inside your Vercel invoice.
+</Admonition>
+
+---
+
+## Installation walkthrough
+
+1. Open the [Neon integration on the Vercel Marketplace](https://vercel.com/marketplace/neon) and click **Install**.
+2. This installs the Native Integration, which automatically creates a Neon account managed by Vercel.
+3. Accept the terms, pick a region & plan, then name your database. (Remember: a "Database" in Vercel is a **Project** in Neon.)
+4. After creation you'll land on Vercel's **Storage** tab that includes status, plan, connection string, billing plan, and more.
+
+### Open the project in the Neon Console
+
+From the Storage tab, click **Open in Neon** to jump straight to your new Neon project dashboard in the Neon Console. You'll notice it lives in an organisation named `Vercel: <your-vercel-team>`.
+
+---
+
+## Connecting the database to a Vercel project
+
+1. In **Storage → `<your database>` → Connect Project** choose the Vercel project and the environments that should receive database variables (Development, Preview, Production).
+
+   ![Connect a Vercel Project](/docs/guides/vercel_native_connect_project.png)
+
+2. (Optional) Under **Advanced Options → Deployments Configuration** enable **Preview** to turn on _Preview Branching_ (see next section).
+
+   ![Vercel deployment configuration](/docs/guides/vercel_native_deployments_configuration.png)
+
+3. Click **Connect**.
+
+<Admonition type="tip" title="Environment variable prefix">You can add a prefix if you have multiple databases in the same project, e.g. `PRIMARY_`.</Admonition>
+
+---
+
+## Enable automated Preview Branching (recommended)
+
+Preview Branching creates an isolated Neon branch (copy-on-write) for every Vercel Preview Deployment so database schema changes can be tested safely.
+
+To enable:
+
+1. While connecting the project (step above) toggle **Required → Preview**.
+2. Make sure **Resource must be active before deployment** is also on. This allows Vercel to wait for the branch to be ready.
+
+Once enabled, the flow looks like this:
+
+1. Developer pushes to feature branch → Vercel kicks off Preview Deployment.
+2. Vercel sends a webhook to Neon → Neon creates branch `preview/<git-branch>`.
+3. Connection string for that branch is injected into the deployment's env vars.
+4. (Optional) Run migrations in build step so schema matches code.
+
+   ![Vercel build commands](/docs/guides/vercel_build_command.png)
+
+Branches live until the git branch is deleted. Enable **Automatically delete obsolete Neon branches** in the **Branches** tab of the integration drawer to delete Neon branches automatically.
```

Comment:
This is just for the Neon-managed integration, right?

Reply:
Oh my, yes! Removed, also this difference is now highlighted in the Vercel overview table.